source/reference/system-variables.txt

```diff
@@ -0,0 +1,37 @@
+.. _system-environment-variables:
+
+----------------
+System Variables
+----------------
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+|bi-short| provides the following environment variables:
+
+.. include:: /includes/fact-system-vars.rst
+
+Setting System Variables
```

Comment:
It's important to specify here that when you set a global variable, it only kicks into effect for _new_ connections - existing connections still maintain the old value. If you sent the session variable instead, it kicks in immediately.

Reply:
Fixed, but I didn't mention 'session variable' since we haven't introduced that concept yet.